src/pages/settings/PasswordPage.js

```diff
@@ -1,20 +1,152 @@
+    }
+
+    handleChangePassword() {
+        changePassword(this.state.currentPassword, this.state.newPassword)
+            .then((response) => {
+                if (response.jsonCode === 200) {
+                    Navigation.navigate(ROUTES.SETTINGS);
+                }
+            });
+    }
 
-const PasswordPage = () => (
-    <ScreenWrapper>
-        <HeaderWithCloseButton
-            title="Change Password"
-            shouldShowBackButton
-            onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
-            onCloseButtonPress={() => Navigation.dismissModal()}
-        />
-    </ScreenWrapper>
-);
+    render() {
+        return (
+            <ScreenWrapper>
+                <>
+                    <HeaderWithCloseButton
+                        title="Change Password"
+                        shouldShowBackButton
+                        onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                        onCloseButtonPress={Navigation.dismissModal}
+                    />
+                    <View style={[styles.p5, styles.flex1]}>
+                        <Text style={[styles.mb6, styles.textP]}>
+                            Changing your password will update your password for both your Expensify.com
+                            and Expensify.cash accounts.
+                        </Text>
+                        <View style={styles.mb6}>
+                            <Text style={[styles.mb1, styles.formLabel]}>Current Password*</Text>
+                            <TextInput
+                                secureTextEntry
+                                autoCompleteType="password"
+                                textContentType="password"
+                                style={styles.textInput}
+                                value={this.state.currentPassword}
+                                onChangeText={currentPassword => this.setState({currentPassword})}
+                            />
+                        </View>
+                        <View style={styles.mb6}>
+                            <Text style={[styles.mb1, styles.formLabel]}>New Password*</Text>
+                            <TextInput
+                                secureTextEntry
+                                autoCompleteType="password"
+                                textContentType="password"
+                                style={styles.textInput}
+                                value={this.state.newPassword}
+                                onChangeText={newPassword => this.setState({newPassword})}
+                                onFocus={() => this.setState({isPasswordRequirementsVisible: true})}
+                                onBlur={() => this.setState({isPasswordRequirementsVisible: false})}
+                            />
+                            {this.state.isPasswordRequirementsVisible && (
+                                <Text style={[styles.textMicro, styles.colorMuted, styles.mt1]}>
```

Comment:
@michelle-thompson @Maftalion @NikkiWines I think my suggestion here would be to create a new style like:

```
formHint: {
        color: themeColors.textSupporting,
        fontSize: variables.fontSizeLabel,
        lineHeight: 18,
    },
```

Also note that in my suggestion above, I am suggesting that we use `fontSizeLabel` which is 13px - I personally feel like the 11px size is too small.